stratum/hal/bin/barefoot/build-bf-sde.sh

```diff
@@ -139,13 +139,31 @@ export SDE_INSTALL=$SDE/install
 # Extract the SDE
 tar xf $SDE_TAR -C $SDE --strip-components 1
 
+# Get SDE version from bf-sde-[version].manifest
+SDE_VERSION=$(find "$SDE" -name 'bf-sde-*.manifest' | head -n 1 | grep -E -e '([0-9]+)\.([0-9]+)\.([0-9]+)' -o)
+if [ -z "${SDE_VERSION}" ]; then
+    echo "Unknown SDE version, cannot find SDE manifest file"
+    exit 1
+else
+    echo "SDE version: ${SDE_VERSION}"
+fi
```

Comment:
There is this `bf-drivers-9.7.0/VERSION` file in the extracted packages. Also contains the just `9.7.0`.
But I'm not sure if that's easier than this.

Reply:
This step is run before p4studio extract all packages, so we need to add additional steps to extract and get VERSION file

```bash
TMP=$(mktemp -d)
find "$SDE" -name bf-drivers-*.tgz | xargs tar -x -C "$TMP" -f
SDE_VERSION=$(cat $TMP/*/VERSION)
```

However, I found another way that might be simpler

```bash
SDE_VERSION=$(find "$SDE" -name 'bf-sde-*.manifest' -printf '%f')
# Remove prefix
SDE_VERSION=${SDE_VERSION#bf-sde-}
# Remove suffix
SDE_VERSION=${SDE_VERSION%.manifest}
```

Reply:
Second way looks good! Technically it's also the "main" SDE version and not the bf-drivers version.